tools/configure.sh

```diff
@@ -277,6 +277,10 @@ if [ -z "${appdir}" ]; then
 
   if [ -d "${TOPDIR}/../apps" ]; then
     appdir="../apps"
+  elif [ -d "${TOPDIR}/../nuttx-apps" ]; then
+    appdir="../nuttx-apps"
+  elif [ -d "${TOPDIR}/../nuttx-apps.git" ]; then
+    appdir="../nuttx-apps.git"
```

Comment:
I think normally when people clone a .git repository the ending ".git" is not created, see:

```
$ git clone https://github.com/apache/nuttx-apps.git
```

Reply:
Yes :-) Two options are added :-) I always add `.git` suffix to easily know what is a git repo and what is not :-P In most cases `../nuttx-apps` will be used :-)